envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/server/callbacks/MetricsDiscoveryServerCallbacks.kt

```diff
@@ -34,9 +36,9 @@ class MetricsDiscoveryServerCallbacks(private val meterRegistry: MeterRegistry)
             .map { type -> type to AtomicInteger(0) }
             .toMap()
 
-        meterRegistry.gauge("grpc.all-connections", connections)
+        meterRegistry.gauge("grpc.connections", Tags.of("type", "all"), connections)
         connectionsByType.forEach { (type, typeConnections) ->
-            meterRegistry.gauge("grpc.connections.${type.name.toLowerCase()}", typeConnections)
+            meterRegistry.gauge("grpc.connections", Tags.of("type", type.name.lowercase()), typeConnections)
```

Comment:
it would be nice to have a connections{protocol="grpc} instead of grpc_connections

Reply:
and apply it to rest of grpc related metrics

Reply:
yea good point, i changed to connections